build.gradle

```diff
@@ -56,14 +56,14 @@ dependencies {
     implementation("com.google.code.gson:gson:2.8.6")
     implementation("com.google.guava:guava:30.1.1-jre")
     implementation(objectify)
-    implementation("com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20200615.1")
+    implementation("com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20211018.2")
     implementation("com.mailjet:mailjet-client:4.5.0")
     implementation("com.sendgrid:sendgrid-java:4.6.0")
     implementation("com.sun.jersey:jersey-client:1.19.4")
     implementation("com.sun.jersey:jersey-core:1.19.4")
     implementation("com.sun.jersey.contribs:jersey-multipart:1.19.4")
     implementation("javax.servlet:javax.servlet-api:3.1.0")
-    implementation("org.apache.solr:solr-solrj:8.8.1")
+    implementation("org.apache.solr:solr-solrj:8.11.0")
```

Comment:
As mentioned before, let's keep this to just `8.8.2` (or any later `8.8.x`) for now. We don't want to deal with any possible regression from `8.8.x` to `8.11.0` for now. Unless there are other vulnerabilities that are only fixed in later versions.

Reply:
mvnrepository was out of date so I didn't see this.
https://solr.apache.org/security.html#apache-solr-affected-by-apache-log4j-cve-2021-44228
Apache Solr releases prior to 8.11.1 were using a bundled version of the Apache Log4J library vulnerable to RCE. For full impact and additional detail consult the Log4J security page.
I think you want to take 8.11.1
I'll do whatever you prefer, 8.8.2 or 8.11.1 but I think you want to take 8.11.1 and retest for possible regression issues.

Reply:
I checked in 8.11.1 that clears log4shell.  I suggest regression testing and getting past log4shell.

Reply:
Actually, it doesn't really matter what version is used here. Here is only the client library; the Solr instance itself is somewhere else that is out of reach from contributors and even core developers.
If we do want to update to `8.11.1`, however, we need to do a bit of regression testing. Let's try by also updating https://github.com/TEAMMATES/teammates/blob/master/solr/Dockerfile on top of the version number of the library and see how it goes from there.